preet-resume.html

```diff
@@ -0,0 +1,123 @@
+      <h1 id='pagetitle'>Preet Gujral - Resume</h1>
+  </header>
+
+
+  <main>
+
+<!--Meat of the resume - Professional Summary, Work Experience, Education-->
+    <section id='profsum' class='content'>
+      <h2 class='sectionhead'>Professional Summary</h2>
+        <p>Full stack javascript developer with a background in customer relationship management, sales, and marketing strategy. Experience working for startups and client-driven professional service organizations. Knowledgable in node, express, behavioral driven development, jquery, and deployment processes such as continuous integration. Goal: contribute to a process-driven dev team for an organization that believes in creating a positive company culture.</p>
+    </section>
+
+    <section id='skills' class='content'>
+      <h2 class='sectionhead'>SKILLS</h2>
+        <p>Full stack javascript developer with a background in customer relationship management, sales, and marketing strategy. Experience working for startups and client-driven professional service organizations. Knowledgable in node, express, behavioral driven development, jquery, and deployment processes such as continuous integration. Goal: contribute to a process-driven dev team for an organization that believes in creating a positive company culture.</p>
+    </section>
+
+    <section id='workexp' class='content'>
+    <h2 class='sectionhead'>Experience</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>Company and Location</th>
+            <th scope="col">Job Role</th>
+            <th scope="col">Responsibilities</th>
+            <th scope="col">Accomplishments</th>
+            <th scope="col">Dates</th>
+          </tr>
+        </thead>
+
+        <tbody>
+
+          <tr>
+            <th scope="col">Code Fellows, Portland, OR</th>
+              <td>Student - Advanced Javascript</td>
+              <td>Responsibilities go here</td>
+              <td>List out some relevant projects</td>
+              <td>Dates attended</td>
+          </tr>
+
+          <tr>
+            <th scope="col">Kalki Consulting, New York, NY</th>
+              <td>Business Development Director</td>
+              <td>Responsibilities go here</td>
+              <td>Accomplishments</td>
+              <td>Dates</td>
+          </tr>
+                    <tr>
+            <th scope="col">GFI Insurance, New York, NY</th>
+              <td>Title</td>
+              <td>Responsibilities go here</td>
+              <td>Accomplishments</td>
+              <td>Dates</td>
+          </tr>          <tr>
+            <th scope="col">Frenkel & Co, New York, NY</th>
+              <td>Title</td>
+              <td>Responsibilities go here</td>
+              <td>Accomplishments</td>
+              <td>Dates</td>
+          </tr>
+        </tbody>
+      </table>
+
+    </section>
+
+    <section>
+      <h2 id='education' class='content'>Education</h2>
+        <h4>Code Fellows, Portland, OR</h4>
+          <h5>dates</h5>
```

Comment:
@preetgujral these are not nested inside anything other than the <section> and should be indented accordingly.  feel free to grab me if you have questions about this